2.7/README.md

```diff
@@ -117,6 +117,29 @@ RUN /usr/libexec/s2i/assemble
 # Set the default command for the resulting image
 CMD /usr/libexec/s2i/run
 ```
+
+If you decide not to use the Source-to-Image scripts, you will need to manually tailor the Dockerfile to your application and its needs.
+Example Dockerfile for a simple Django application:
+
+```
+FROM registry.access.redhat.com/ubi8/python-27
+
+# Add application sources
+USER 0
```

Comment:
Although we don't have it documented in other images either, we might document here why we change USER to 0 and group for the app files -- that it allows to run the resulting image as any user in OpenShift.

Reply:
During testing, I've just realized that we don't need to switch users to 0 and back at all because all we do has an impact in user-writable folders in the virtual environment.

What are you probably talking about is documented [here](https://github.com/sclorg/s2i-python-container/blob/master/src/Dockerfile.template#L85) and it's achieved by our scripts so we either skip this step here as not needed or add and document it.

Reply:
I'm gonna try it without it and we'll what openshift tests will say,

Reply:
I am not sure I follow here. The `USER 0` was needed so that the files copied into the new layer have group set to `0` (since Openshift is run with a random uid user that is member of the 0 group).

>What are you probably talking about is documented here and it's achieved by our scripts so we either skip this step here as not needed or add and document it.

Which scripts make this not an issue for the Dockerfile example?

Reply:
That said, would this (non-s2i) example even be aimed at Openshift use?

Reply:
Do I understand it correctly that we have to switch to user 0, copy files, change ownership and switch batch to user 1001 in all our examples? Is the fix-permissions needed here after the commands?